api/src/main/java/org/apache/iceberg/actions/RewriteStrategy.java

```diff
@@ -71,8 +71,9 @@
    * Method which will rewrite files based on this particular RewriteStrategy's algorithm.
    * This will most likely be Action framework specific (Spark/Presto/Flink ....).
    *
+   * @param setId an identifier for this set of files
```

Comment:
Since we are rewriting files in groups (if I'm not mistaken), would it make more sense to refer to this as a `groupId`?

Reply:
Sure I was mostly copying the current usage in RewriteManager